st2common/bin/st2ctl

```diff
@@ -173,10 +178,10 @@ function clean_logs() {
 
 function getpids() {
   echo "##### st2 components status #####"
-  COMPONENTS=${COMPONENTS/mistral/mistral-server mistral-api}
+  COMPONENTS=${COMPONENTS/mistral/mistral-server mistral.api}
```

Comment:
Just a note:
`mistral.api` showing up for st2ctl status could be confusing, considering that the service name in `mistral-api` (eg. `service mistral-api`), but this probably minor and could be ignored for simplicity.

Reply:
Yeah, I know. No particularly easy way around it if we're going to rely on process name for checking status though. Better way is probably to capture PID at start time, store in `/var/run`, and check that.